index.html

```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
             </nav>
         </header>
         <main>
-            <section>
+            <div class="main">
+                <section id="about-me">
+                    <div>
+                        <div>
+                            <h1>
+                                I'm William
+                                Glad to see you!
+                            </h1>
+                        </div>
+                        <div class="description">
+                            <p>I'm a software developer! I can help you build a product , feature or website Look
+                                through
+                                some
+                                of my
+                                work and
```

Comment:
It is a requirement that you make use of a CSS grid for your layout in this section, I can see you made use of flexbox which is cool. but this milestone requires you make use of grid to make your skills with CSS grid stronger🤗 kindly make the right changes🤗